.github/workflows/pack-nsis.yml

```diff
@@ -142,3 +155,13 @@ jobs:
           makeLatest: false
           prerelease: true
           updateOnlyUnreleased: true
+      - name: Upload KNI zip to the release
+        uses: ncipollo/release-action@v1.14.0
+        with:
+          allowUpdates: true
+          artifacts: KNI-*.zip
+          body: '**For testing purposes only**'
+          draft: false
+          makeLatest: false
+          prerelease: true
+          updateOnlyUnreleased: true
```

Comment:
I think you can use just one release action with:

`artifacts: "khiops-*-setup.exe,KNI-*.zip"`

As specified in the [action's docs](https://github.com/ncipollo/release-action?tab=readme-ov-file#example)

Reply:
done